src/patterns/dates/index.md.njk

```diff
@@ -82,9 +82,11 @@ To check that the dates users give you are valid, make sure that:
 - past dates are in the past
 - future dates are in the future
 - the first date in a date range is before the second
+- a date with or without a leading zero is accepted
 
 ### How to write dates
 
+Do not use a leading zero in example dates. If you provide an example day number, use a number greater than 12. If you provide an example month number, use a number greater than 12. This will emphasise the distinction between the day field and the month field which can be a problem for some users, particularly if accustomed to other date formats.
```

Comment:
I don't think this is quite right – it says that both day and month should be greater than 12.

```suggestion
Do not use a leading zero in example dates. If you provide an example day number, use a number greater than 12. If you provide an example month number, use a number less than or equal to 12. This will emphasise the distinction between the day field and the month field which can be a problem for some users, particularly if accustomed to other date formats.
```